lisa/operating_system.py

```diff
@@ -794,6 +800,13 @@ def _install_packages(
         install_result = self._node.execute(
             command, shell=True, sudo=True, timeout=timeout
         )
+        if self._debian_package_install_unauthenticated_regex.search(
```

Comment:
If we allow this blindly, it can be added by default. So that would be succeeded at first try.

Reply:
updated